protonmail/attachments.go

```diff
@@ -173,7 +190,16 @@ func (c *Client) CreateAttachment(att *Attachment, r io.Reader) (created *Attach
 			return
 		}
 
-		// TODO: Signature
+		// TODO use Signature: string here
+		if att.SignatureBuff.Len() != 0 {
+			if w, err := mw.CreateFormFile("Signature", "blob"); err != nil {
+				pw.CloseWithError(err)
+				return
+			} else if _, err := io.Copy(w,  bufio.NewReader(&att.SignatureBuff)); err != nil {
```

Comment:
`io.WriteString(w, att.Signature)` should do the trick.

Reply:
Thank you, that does the trick